python/pyspark/pandas/indexes/base.py

```diff
@@ -527,6 +527,8 @@ def map(
         """
         Map values using input correspondence (a dict, Series, or function).
 
+        versionadded:: 3.3.0
```

Comment:
We don't need this because `Index.map` was merged to 3.2 as well.

https://github.com/apache/spark/blob/branch-3.2/python/pyspark/pandas/indexes/base.py#L524-L566

Reply:
Oh right.